index.js

```diff
@@ -8,7 +8,8 @@ var cachedDependencies = [];
 
 var defaultOptions = {
   cache: false,
-  yes: true
+  yes: true,
+  warn: true
```

Comment:
I actually dig this idea. It's easy to turn off if you don't like them, and culturally it seems nice to encourage people to clean things up.

@eeue56 any objection to trying it this way and changing it if it's annoying in practice?